src/components/calcite-color-picker-hex-input/calcite-color-picker-hex-input.tsx

```diff
@@ -257,8 +248,9 @@ export class CalciteColorPickerHexInput {
           class={CSS.input}
           dir={elementDir}
           label={intlHex}
+          maxLength={6}
           onCalciteInputBlur={this.onCalciteInputBlur}
-          onChange={this.onInputChange}
+          onCalciteInputChange={this.onInputChange}
```

Comment:
@jcfranco seems like this may be an existing bug?

Reply:
Seems like we should also use `onCalciteInputInput` here as well.

Reply:
maybe the internal onChange is bubbling up or something, seems to work on prd.

Reply:
Marking as resolved since your changes greatly simplify this. 🎉